lib/tasks/argo.rake

```diff
@@ -79,6 +97,23 @@ namespace :argo do
     desc "Overwrite Solr configs and JARs"
     task :config => %w[argo:solr:config] do   # TODO: argo:fedora:config
     end
+
+    # Note: this rake task is failing, with error:
+    # Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8983
+    #
+    # desc "Start Jetty with Fedora/Solr checks"
+    # task :start do
+    #   Rake::Task['jetty:restart'].invoke
+    #   fedora_url = 'http://localhost:8983/fedora/'
+    #   puts "Checking Solr"
+    #   if web_response_ok? 'http://localhost:8983/solr/'
+    #     puts "Success! Solr responds"
+    #   end
+    #   if web_response_ok? 'http://localhost:8983/fedora/'
+    #     puts "Success! Fedora responds."
+    #   end
+    # end
+
```

Comment:
would be in favor of dropping both these commented out sections, since i generally lean towards not committing commented out code.  also, if we did keep this, i'd want to use the configured fedora and solr URLs instead of hardcoding the URL (even if we pretty much always expect localhost on laptop, the port could change).

what's the intended behavior of the commented out method and task?